test_scripts/Defects/8_2/2468/common.lua

```diff
@@ -0,0 +1,209 @@
+local common = require('test_scripts/Smoke/commonSmoke')
```

Comment:
```suggestion
---------------------------------------------------------------------------------------------------
-- Common module
---------------------------------------------------------------------------------------------------
--[[ Required Shared libraries ]]
local common = require('test_scripts/Smoke/commonSmoke')
```
Just for consistency